microsoft-dynamics-365/2.4/modules/ROOT/pages/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -0,0 +1,930 @@
+ | The connection types that can be provided to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> | Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. | |
+| Encoding a| | The default body encoding type. If the encoding type is not specified at the operation level, this encoding type is used. | |
+|===
+
+==== Connection Types
+
+[[dynamics-config_connection]]
+===== Connection
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Version a| String |  The version number of the API to which to connect |  9.0 |
+| Scope a| String |  |  openid |
+| Resource a| String | Connector resource URL |  | x
+| Response Type a| String |  |  query |
+| Read Timeout a| Number |  Specifies the amount of time, in milliseconds, that the consumer waits for a response before it times out. Default value is 0, which means infinite. |  0 |
+| Connection Timeout a| Number |  Specifies the amount of time, in milliseconds, for which the consumer attempts to establish a connection before the connection times out. Default value is 0, which means infinite. |  0 |
+| Host a| String |  Hostname of the proxy. If this property is not set, then no proxy will be used; otherwise, a proxy will be used, but a proxy host must be specified. |  |
+| Port a| Number |  Port of the proxy. If host is set, then this property must be set and cannot be a negative number. |  |
+| Username a| String |  Username used to authenticate against the proxy. If this property is not set, then  authentication is not applied to the proxy, otherwise this value must be specified. |  |
+| Password a| String |  Password used to authenticate against the proxy. |  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+| Pooling Profile a| <<PoolingProfile>> |  Characteristics of the connection pool |  |
+| Consumer Key a| String |  The OAuth key registered with the service provider |  | x
+| Consumer Secret a| String |  The OAuth consumer secret registered with the service provider |  | x
+| Authorization Url a| String |  The service provider's authorization endpoint URL |  `+https://login.microsoftonline.com/oauth2/authorize+` |
+| Access Token Url a| String |  The service provider's accessToken endpoint URL |  `+https://login.microsoftonline.com/oauth2/token+` |
+| Scopes a| String |  The OAuth scopes to be requested during the dance. If not provided, this value defaults to the scopes in the annotation. |  |
+| Resource Owner Id a| String |  The ID each component should use if it doesn't have another reference. |  |
```

Comment:
What is "it"? 

The ID each component should use if the component doesn't have a reference to a resource owner ID? 

I'm not sure.